MaterialDesignThemes.Wpf/BindingEx.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Windows.Data;
+
+namespace MaterialDesignThemes.Wpf
+{
+    internal static class BindingEx
```

Comment:
This library typically names these classes/files with a suffix of "Extensions", so I think this should be `BindingExtensions`

Reply:
Done in latest commit